meta/main.yml

```diff
@@ -17,5 +17,5 @@ galaxy_info:
     - openstack
 
 dependencies:
-  - role: stackhpc.os-shade
-    os_shade_venv: "{{ os_networks_venv }}"
+  - role: stackhpc.os_openstacksdk
+    os_openstacksdk_venv: "{{ os_networks_venv }}"
```

Comment:
Should we added upper constraints like in some other repos?

Reply:
Could do. I think we only have it for os-images since it installs its own packages. For other roles I usually specify the shade/openstacksdk/openstackclient UC variable directly.

Reply:
Would be a separate change though I think.